otx/algorithms/segmentation/adapters/mmseg/models/utils/aggregator.py

```diff
@@ -1,21 +1,26 @@
-# Copyright (C) 2022 Intel Corporation
+"""Aggregators."""
+# Copyright (C) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
 
 import torch
-import torch.nn as nn
 import torch.nn.functional as F
 from mmcv.cnn import ConvModule, DepthwiseSeparableConvModule
+from torch import nn
 
-from ..utils import normalize
+from otx.algorithms.segmentation.adapters.mmseg.models.utils import normalize
 
 
+# pylint: disable=invalid-name
```

Comment:
Please disabling pylint as minimum as possible. How about setting in on necessary functions or classes?

Reply:
Let me do that later. I leave them as is because I have no idea yet about renaming. I need to see them anyway when I debug mmseg fcnhead.